terraform/ratingservice/configure_ratings_db.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# 
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#--------------------------------------------------------------------------
+# Environment variables:
+# DBHOST - IP address of the Cloud SQL instance
+# DBNAME - a name of the Postgres DB
+# DBUSER - DB user
+# DBPWD  - DB user password
+#--------------------------------------------------------------------------
+
+#
+# authorize access to sql instance from local machine
+# (machine has to have public ip)
+#
+MY_PUBLIC_IP=`dig TXT +short o-o.myaddr.l.google.com @ns1.google.com | awk -F'"' '{ print $2}'`
+gcloud sql instances patch $INSTANCE_NAME --authorized-networks=$MY_PUBLIC_IP --quiet
+
+#
+# configure db schema and populate rating entities
+#
+echo "Creating table [ratings]..."
+PGPASSWORD=$DBPWD psql -U $DBUSER -h $DBHOST $DBNAME <<EOF
+CREATE TABLE IF NOT EXISTS ratings(
+    eid char(16),
+    rating real,
+    votes integer,
+    PRIMARY KEY(eid)
+);
+CREATE TABLE IF NOT EXISTS votes(
+    id SERIAL,
+    eid char(16),
+    rating integer,
+    in_process boolean DEFAULT FALSE,
```

Comment:
why do we need this?

Reply:
do you ask about the table [votes] or about last field of the table? the table is used to collect new votes following CQRS model of request handling which decouples store and read operations. the last field is used to mark records that are in the process of being aggregated and added to the [ratings] from new coming new votes. this process is also described in the [TDD](https://docs.google.com/document/d/1k5Zq1BEyxhVe7Ko3_4dlLxhMXl4GC_tBp-6g1nOdnPg) that is referenced in the PR description.